src/sentry/utils/appleconnect/appstore_connect.py

```diff
@@ -138,7 +138,7 @@ def _get_appstore_json(
         full_url += url
         logger.debug(f"GET {full_url}")
         with sentry_sdk.start_span(op="http", description="AppStoreConnect request"):
-            response = session.get(full_url, headers=headers, timeout=REQUEST_TIMEOUT)
+            response = session.get(full_url, headers=headers, timeout=REQUEST_TIMEOUT * 2)
```

Comment:
This is the only place in which we use `REQUEST_TIMEOUT`.  Might as well change the value directly.

I'd also be tempted to change the line above to `with sentry_sdk.start_transaction(op='http', description='AppStoreConnect request'):` so that we get a searchable transaction which we can then see how long it usually takes, what the outliers are etc.